### ISSUE TITLE
Issues loading spectrum $element.find(...).spectrum is not a function

### ISSUE THREAD

**Issue by imoore** [YourITGroup/UmbColourPicker#3]
I started using the Umbraco Plugin for spectrum and it was all working fine.

![umbraco_spectrum_working](https://user-images.githubusercontent.com/4741003/41027391-d23e6edc-696e-11e8-9e4b-00d3ca9eea0c.PNG)

the next day I was on the back-end and tried to change the color property and this was happening

![umbraco_spectrum_not_working](https://user-images.githubusercontent.com/4741003/41027434-f64d03c4-696e-11e8-8315-7fc4ed4ba107.PNG)

The console log was showing this error

```js
TypeError: $element.find(...).spectrum is not a function
    at VM87 SpectrumColorPicker.controller.js:42
    at i (VM24 angular.min.js:79)
    at VM24 angular.min.js:80
    at Object.$eval (VM24 angular.min.js:92)
    at Object.$digest (VM24 angular.min.js:90)
    at Object.$apply (VM24 angular.min.js:92)
    at HTMLDivElement.<anonymous> (VM29 angular-mobile.js:269)
    at HTMLDivElement.dispatch (VM23 jquery.min.js:3)
    at HTMLDivElement.r.handle (VM23 jquery.min.js:3) 
Failed to load resource: the server responded with a status of 404 (Not Found) 
http://localhost:50574/~/App_Plugins/SpectrumColorPicker/lib/spectrum/spectrum.js?umb__rnd=263005eabbbd1acfda215647b6215176 
```
I changed the ```assetsService.loadJS``` from

```js
        assetsService.loadJs(
            "~/App_Plugins/SpectrumColorPicker/lib/spectrum/spectrum.js"
        ).then(function () {
```

to 

```js
        assetsService.loadJs(
            "/App_Plugins/SpectrumColorPicker/lib/spectrum/spectrum.js"
        ).then(function () {
```

And everything was working again

**Comment by robertjf** [comment]
Hey, what version of Umbraco are you using this with?

**Comment by imoore** [comment]
Hi I'm currently using it with 
`Umbraco version 7.7.6 assembly: 1.0.6528.26344`

**Comment by prjseal** [comment]
Fix submitted in this PR #6

**Comment by robertjf** [comment]
closed by @prjseal PR :)
